web/templates/input_pill.hbs

```diff
@@ -7,6 +7,9 @@
         {{~#if has_status~}}
             {{~> status_emoji status_emoji_info~}}
         {{~/if~}}
+        {{#if is_bot}}
```

Comment:
Use tildes here (`~`) to collapse the whitespace, as in the `has_status` if statement above, and then you shouldn't have to modify the `search.test.js` file as you've done here.

Reply:
Updated...